traversal/planner/GraphPlanner.java

```diff
@@ -262,18 +261,16 @@ private void optimise(GraphManager graphMgr, boolean singleUse) {
         // TODO: we should have a more clever logic to allocate extra time
         long allocatedDuration = singleUse ? HIGHER_TIME_LIMIT_MILLIS : DEFAULT_TIME_LIMIT_MILLIS;
         Instant start, endSolver, end;
-        totalDuration += allocatedDuration;
 
         start = Instant.now();
-        optimiser.optimise(totalDuration);
+        optimiser.optimise(allocatedDuration);
```

Comment:
we find that the SAT solver always restarts so we don't need to increase the allocated duration over multiple invocations

Reply:
And so we don't need to keep the variable at all.